haechan45th/src/routes/index.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const router = express.Router();
+
+const userRouter = require("./userRouter");
+const postRouter = require("./postRouter");
+const likeRouter = require("./likeRouter");
```

Comment:
```suggestion
const express = require("express");

const userRouter = require("./userRouter");
const postRouter = require("./postRouter");
const likeRouter = require("./likeRouter");

const router = express.Router();
```
필요한 모듈들을 각각 호출하고 이를 활용하는 코드 순으로 작성될 수 있게 순서를 수정해주세요.